src/vbk/rpc_register.cpp

```diff
@@ -588,11 +584,10 @@ UniValue getrawpayload(const JSONRPCRequest& request, const std::string& name)
         } else {
             result.pushKV("confirmations", 0);
         }
+        result.pushKV("blockhash", blockindex->GetBlockHash().GetHex());
     }
 
     result.pushKV(name, altintegration::ToJSON<UniValue>(out));
-    result.pushKV("blockhash", hash_block.GetHex());
```

Comment:
pls do not remove this, otherwise it will be API break

Reply:
if block is not provided, and multiple containing blocks are returned by index, then return one that is on main chain. 
if payload is not on main chain, in this field return 0000...000

also add new field, `containing_blocks: [hash, hash, hash]` which will be just a list of hashes of vBTC blocks containing that payload (returned from storage index)